src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -351,7 +350,7 @@ public String getQuestionAdditionalInfoHtml(int questionNumber,
     public String getQuestionResultStatisticsHtml(
             List<FeedbackResponseAttributes> responses,
             FeedbackQuestionAttributes question,
-            PageData pageData,
+            String studentEmail,
```

Comment:
I was thinking along the lines of introducing separate abstract methods for instructors and students, where the instructor's method basically wraps the student's method. Right now, that <code>studentEmail</code> should always be null for instructorResults, so it looks a little odd in the code.